frontend/__tests__/components/LikeList.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { render, screen } from '@testing-library/react'
+import LikeList from '@/components/LikeList'
+
+const FAKE_RESTAURANT_ONE = {
+  name: 'Fake Restaurant 1',
+  address: '123 Fake St.',
+  cuisine: 'Fakish',
+  rating: 4.5,
+  phone: '2130001111',
+  image_url: 'http://www.dummyrestaurant.com/pic',
+  website_url: 'http://www.dummyrestaurant.com',
+}
+
+const FAKE_RESTAURANT_TWO = {
+  name: 'Fake Restaurant 2',
+  address: '123 Fake St.',
+  cuisine: 'Asian',
+  rating: 4.0,
+  phone: '2130001111',
+  image_url: 'http://www.dummyrestaurant.com/pic',
+  website_url: 'http://www.dummyrestaurant.com',
+}
+
+const FAKE_RESTAURANT_THREE = {
+  name: 'Fake Restaurant 3',
+  address: '123 Fake St.',
+  cuisine: 'American',
+  rating: 3.5,
+  phone: '2130001111',
+  image_url: 'http://www.dummyrestaurant.com/pic',
+  website_url: 'http://www.dummyrestaurant.com',
+}
+
+const LIKED_RESTAURANTS = [FAKE_RESTAURANT_ONE, FAKE_RESTAURANT_TWO, FAKE_RESTAURANT_THREE]
+
+describe('LikeList', () => {
+  it('renders the TasteMate h1', () => {
+    render(<LikeList likedRestaurants={[]} />)
+    const myElem = screen.getByText('TasteMate')
+
+    expect(myElem).toBeInTheDocument()
```

Comment:
nitpick:  

Generally, React Testing Library emphasizes writing tests that align closely with user interactions and experience, ie. testing what the user actually sees and how they may interact with the UI.

It's not officially in the docs anywhere but given this philosophy, I usually prefer to use `.toBeVisible` over `.toBeInTheDocument` as it's more idiomatic and is more from the user perspective. As example of why this matters is that an element could be "in the document" but not "be visible." (More info [here](https://github.com/testing-library/jest-dom#tobevisible) on other things it accounts for.)

Can we change these to be `toBeVisible`?

Also a quick tip: The only time I really using `.toBeInTheDocument` is when I `queryBy` something to `.not.toBeInTheDocument`. Or if I want it test that it's in the document but might not be visible to the user.